pkgs/applications/window-managers/dwm/dwm-status.nix

```diff
@@ -15,11 +20,17 @@ rustPlatform.buildRustPackage rec {
     dbus
     gdk_pixbuf
     libnotify
+    makeWrapper
     pkgconfig
```

Comment:
while we are at it lets move `pkgconfig` to `nativeBuildInputs`, also `makeWrapper` should be a `nativeBuildInput`.